docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -0,0 +1,320 @@
+# Kubermatic 2.26
+
+- [v2.26.0](#v2260)
+
+
+## v2.26.0
+
+**GitHub release: [v2.26.0](https://github.com/kubermatic/kubermatic/releases/tag/v2.26.0)**
+
+Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.26/installation/upgrading/). Consider tweaking `seedControllerManager.maximumParallelReconciles` to ensure user cluster reconciliations will not cause resource exhaustion on seed clusters. A [full upgrade guide is available from the official documentation](https://docs.kubermatic.com/kubermatic/v2.25/installation/upgrading/upgrade-from-2.25-to-2.26/).
+
+### Action Required
+
+- Update to controller-runtime 0.19 / Kubernetes 1.31 dependencies ([#13621](https://github.com/kubermatic/kubermatic/pull/13621))
```

Comment:
All the action required items do not each need a big "ACTION REQUIRED" prefix, I think we can safely get rid of them.